plugin.onedrive/addon.py

```diff
@@ -278,17 +271,17 @@ def close_dialog_timeout(dialog, timeout):
         current_time = time.time()
     dialog.close()
 
-def export_folder(name, item_id, driveid, destination_folder, base_folder, directLink=None):
+def export_folder(basename, item_driveid, item_id, driveid, destination_folder, base_folder, directLink=None):
     onedrive = get_onedrive(driveid)
-    parent_folder = os.path.join(destination_folder, name)
-    if not os.path.exists(parent_folder):
+    parent_folder = destination_folder + basename + '/'
```

Comment:
can you please use os.path.join here? It was my only comment before. Also you don't need to close and open another pull request. Just add more commits to your branch `guZZman:helix`